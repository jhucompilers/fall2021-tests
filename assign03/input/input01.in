PROGRAM add;
  VAR a, b, sum: INTEGER;

BEGIN
  READ a;
  READ b;
  sum := a + b;
  WRITE sum;
END.
