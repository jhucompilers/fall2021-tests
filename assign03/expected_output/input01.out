0,VAR,a,INTEGER
0,VAR,b,INTEGER
0,VAR,sum,INTEGER
